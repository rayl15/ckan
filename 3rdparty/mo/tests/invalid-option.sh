#!/usr/bin/env bash
# This should display a message indicating that the file --something
# could not be found.
cd "${0%/*}"
../mo --something 2>&1
