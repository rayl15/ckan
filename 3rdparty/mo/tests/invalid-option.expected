cat: --something: No such file or directory
